stack/__main__.py

```diff
@@ -144,21 +144,31 @@ def bucket_member(*args, **kwargs):
         opts=pulumi.resource.ResourceOptions(depends_on=[cloudidentity]),
     )
 
-    main_upload_bucket = create_bucket(
-        bucket_name('main-upload'), lifecycle_rules=[undelete_rule]
-    )
+    main_upload_buckets = {
+        'main-upload': create_bucket(
+            bucket_name('main-upload'), lifecycle_rules=[undelete_rule]
+        )
+    }
+
+    for additional_upload_bucket in (
+        config.get_object('additional_upload_buckets') or ()
+    ):
+        main_upload_buckets[additional_upload_bucket] = create_bucket(
+            additional_upload_bucket, lifecycle_rules=[undelete_rule]
```

Comment:
Should `additional_upload_bucket` be wrapped in `bucket_name(additional_upload_bucket)`?

Reply:
I thought about this, but then you're always restricted to `cpg-{dataset}-...`, which  felt a little limiting, what do you think?